api/build.gradle

```diff
@@ -8,6 +8,7 @@ plugins {
 group = 'com.code4ro'
 version = '0.0.1-SNAPSHOT'
 sourceCompatibility = '11'
+targetCompatibility = '11'
```

Comment:
Why not 13?

Reply:
source = 11

Reply:
Yes, but we can use Java 13.
sourceCompatibility=13
targetCompatibility=13

Reply:
I can do the change if others agree

Reply:
I think we can live with 11

Reply:
We choose 11 because it's the latest LTS